src/util/routeUtils.js

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
Please move this to `/ui/src/util/urlUtils.js` where we already have similar route/url checks.

Reply:
Sure thing!

Reply:
There is also a test file, `test/unit/specs/util/urlUtils.spec.js` where you could add a simple test for your new method.